azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -625,10 +626,17 @@ private void deleteExecutionDir(final int executionId) {
 
   @Override
   public void handleEvent(final Event event) {
-    if (event.getType() == EventType.FLOW_FINISHED || event.getType() == EventType.FLOW_STARTED) {
-      final FlowRunner flowRunner = (FlowRunner) event.getRunner();
-      final ExecutableFlow flow = flowRunner.getExecutableFlow();
+    // Adding extra logging for debuggability of execution directory cleanup call
```

Comment:
Please add a todo which states this piece of code backs to original state since the structure has changed.

Reply:
Adding the todo

Reply:
Alternatively maybe the entire extra work done here (not just the output resulting from this additional work) could be predicated on logging level: `if (!log.getEffectiveLevel().isGreaterOrEqual(Level.INFO)) { ...`

Reply:
Good point @li-ygerchikov. I would like to keep rest of the logs for future purpose as well but i would like to delete these logs once issue is solved. Is that ok with you?

Reply:
Yes, it's OK. Since you are still logging at INFO level, no objections.